### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Adds/removes/cleans up comments.
- Removes trailing whitespace.
- Updates `bower.json` and `package.json` with current changelog version.

## Review

- @sebworks 
- @virginiacc

### CODE REVIEW

**Comment by sebworks** [comment]
👍

File: drop-db.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # ==========================================================================
-# Setup script for MYSQL Databse Instantiation
+# Setup script for MySQL database instantiation—used to drop the database.
```
Comment (sebworks) [inline drop-db.sh]: Should this just be `Setup script used to drop MySQL database`. It's not as though the script has any knowledge on how it's being used.

Comment (anselmbradford) [inline drop-db.sh]: Fixed!

File: initial-test-data.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # ==========================================================================
-# Initialization script for a wagtail user and imported data
+# Initialization script for a wagtail user and imported data.
```
Comment (sebworks) [inline initial-test-data.sh]: Should this ` Initialization script for a wagtail user and imported test data.`?

**Comment by sebworks** [comment]
💯, merge at will.

File: initial-test-data.sh (hunk 2 above)
Comment (anselmbradford) [inline initial-test-data.sh]: Fixed!

**Pull request merged**